storage.go

```diff
@@ -711,13 +716,18 @@ var (
 	allTablesToDelete = append(tablesToDeleteOCP, tablesToDeleteDVO...)
```

Comment:
please add it here

Reply:
No, don't add it. It has a separated loop because the difference in the `Exec` parameters